src/types-v1.ts

```diff
@@ -38,7 +38,7 @@ export type Error = {
 
 type MessageBase = {
   feed: string
-  chainId: string
+  chainId?: string
```

Comment:
Should we keep this as required? These feeds could be setup for multiple chains in future.

Reply:
@aaronbarnardsound Yep sorry about that added for testing - I dont think chainId serialization is working for any v1 messages. I havent had time to figure out the issue but it is coming through as null for all v1 messages

Reply:
Hmm that's weird. I'll take a look.